test/fixtures/dialects/mariadb/.sqlfluff

```diff
@@ -0,0 +1,2 @@
+[sqlfluff]
+dialect = mysql
```

Comment:
Shouldn't this be mariadb?

Reply:
Yes, I just copied all the mysql tests over to mariadb and kept them as is. Will change this as well